modules/ossm-cert-manager-install-istio-resource.adoc

```diff
@@ -0,0 +1,149 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/install/ossm-cert-manager-assembly.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="install-istio-resource_{context}"]
+= Install your Istio resource
+
+//TP1 content influx. Title, etc may change.
+//Content is very similar to 2.x content
+//all kinds of formatting things to fix. want to see if a build will generate to have a look, and see how it fits structurally with the IA.
+
+
+After you have installed `istio-csr` by following the procedure for either `InPlace` or `RevisionBased`, you can install the `Istio` resource.
+
+You need to disable Istio's built in CA server and tell istiod to use the `istio-csr` CA server. The `istio-csr` CA server issues certificates for both istiod and user workloads.
+
+.Procedure
+
+. Create the `Istio` object as in the following example:
++
+.Example `Istio.yaml` object
```

Comment:
Since the next step is `oc apply -f istio.yaml` it should probably match what is defined here.
```suggestion
.Example `istio.yaml` object
```